crates/re_renderer/src/importer/gltf.rs

```diff
@@ -243,13 +240,19 @@ fn import_mesh(
         anyhow::bail!("empty mesh");
```

Comment:
Really should get rid of all anyhow errors in all of re_renderer. One of the many things I never seem to get to :/

Reply:
I found that copilot was really good at writing `thiserror` `enum`s, so just do it